docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -0,0 +1,208 @@
+---
+sidebar_title: BackgroundTask
+title: Background Task
+description: A library that provides an API for running background tasks.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-background-task'
+packageName: 'expo-background-task'
+platforms: ['android', 'ios']
+isNew: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { PlatformTags } from '~/ui/components/Tag/PlatformTags';
+
+`expo-background-task` provides an API to run deferrable background tasks in a way that optimizes battery and power consumption for the end user. This module uses the [`BGTaskScheduler`](https://developer.apple.com/documentation/backgroundtasks/bgtaskscheduler) API on iOS and the [`WorkManager`](https://developer.android.com/topic/libraries/architecture/workmanager) API on Android to schedule tasks. It also uses the [expo-task-manager](task-manager.mdx) Native API for running JavaScript tasks.
+
+## Background tasks
+
+A background task is a deferrable unit of work that is performed in the background, outside the lifecycle of your app. This is useful for tasks that need to be executed when the app is inactive, such as syncing data with a server, fetching new content, or even checking if there are any [Expo Updates](updates.mdx).
+
+### When are background tasks run?
+
+The Expo Background Task API leverages each platform to execute tasks at the most optimal time for both the user and the device.
+
+This means that the task may not run immediately after it is scheduled, but it will run at some point in the future if the system decides so.
+
+A background task will only run if the battery has enough charge (or the device is plugged into power) and the network is available. Without these conditions, the task won't execute. The exact behavior will vary depending on the operating system.
+
+### When will they stop?
+
+The native platform APIs will stop running background tasks if the user explicitly kills the app (they will be resumed when the user restarts the app) or if the app is uninstalled. If the device is rebooted or the app is killed by the system, the background tasks will be resumed.
+
+> **Info** The non-deterministic behavior of background tasks is intentional, aimed at optimizing battery life and enhancing user experience. If a user force stops an app, the system interprets this as a signal not to run background tasks for that app until the user restarts it. On iOS an app is force stopped if the user swipes up on the app in the app switcher. On Android this behaviour depends on the device vendor. Some vendors have implementations where the app is force stopped when the user removes the app from the recent apps list.
```

Comment:
```suggestion
> **Info** The non-deterministic behavior of background tasks is intentional, aimed at optimizing battery life and enhancing user experience. If a user force stops an app, the system interprets this as a signal not to run background tasks for that app until the user restarts it.On Android, this behavior depends on the device vendor. On iOS, an app is force stopped if the user swipes up on the app in the app switcher.  Some vendors have implementations where the app is force stopped when the user removes the app from the recent apps list.
```

- Let's apply the platform order preference for Android and iOS.
- We prefer to use American English spellings in our docs [reference: Writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#voice-and-tone).